series-tvdb.en.yaml

```diff
@@ -121,6 +121,21 @@ entries:
         anilist-id: 20845
       - season: 4
         anilist-id: 21379
+  - title: "Fate/Stay night: Unlimited Blade Works"
+    seasons:
+      - season: 1
+        anilist-id: 19603
+        start: -1 # Anilist has 26 Episodes (includes Special EP) while TVDB has 25 Episodes (excludes Special EP)
+      - season: 1
+        anilist-id: 20792
+        start: 13
```

Comment:
Does it start at 13? On **AniList**, Season 1 has 13 episodes, which means Season 2 would start on 14?

Reply:
S1 starts at episode 0 instead of episode 1 on Anilist.

Reply:
Alright that kinda makes sense, sometimes it's confusing how the starting points work

Reply:
Episode 0 to 12 (total 13 episodes) are in S1. Episode 0 being **Prologue** and Episode 12 being **The Final Decision** where cour 1 ended. Cour 2 starts at Episode 13 **Time of Departure**.